backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs

```diff
@@ -298,7 +295,7 @@ public static IQueryable<Droit> BuildQuery(DroitSearchForm form, IQueryable<Droi
                     !string.IsNullOrEmpty(d.District) &&
                     (d.District.ToLower().Contains(form.District.ToLower()) || 
                     (usePsql? EF.Functions.FuzzyStringMatchLevenshtein(form.District.ToLower(), d.District.ToLower()) :
-                        SearchHelper.GetLevenshteinDistance(form.District.ToLower(), d.District.ToLower())) < MaxLevenshteinDistance
+                        SearchHelper.GetLevenshteinDistance(form.District.ToLower(), d.District.ToLower())) < 1
```

Comment:
Probably worth checking this, might need to be <=

Reply:
probably as the distance is how many 'changes' there are if the two strings are laid atop each other, so atm these only returns exact matches I believe.

Reply:
Maybe introduce a constant for LowLevenshteinDistance being 2?

Reply:
Could be a good idea, especially for maintaining it down the line